cmd/limactl/template.go

```diff
@@ -147,6 +153,39 @@ func templateCopyAction(cmd *cobra.Command, args []string) error {
 	return err
 }
 
+func newTemplateYQCommand() *cobra.Command {
+	templateYQCommand := &cobra.Command{
+		Use:   "yq TEMPLATE EXPR",
+		Short: "Query template expressions",
+		Long:  "Extract template data using a yq expression",
```

Comment:
Please add how this differs from the standalone `yq` command

Reply:
The only difference is that it works even when you don't have `yq` installed (e.g. on Windows GitHub runners). Not sure how you want to put this into the command help.

This is essentially the same as the `gh api ... --jq` option, having the tool already built in, so you don't need to worry about having it installed.

<hr>

I'm planning to eventually add `--yq` options to `limactl ls` and `limactl info` as well, but not right now.

Reply:
@AkihiroSuda I don't really know what you are looking for. Is this better:

```
Use builtin YQ evaluator to extract information from a template.
The template has embedded all external references and default values
are filled in before the YQ expression is evaluated.
```

Reply:
I've updated the PR to produce the following output:

```console
❯ limactl tmpl yq --help
Use the builtin YQ evaluator to extract information from a template.
External references are embedded and default values are filled in
before the YQ expression is evaluated.

Example:
    limactl template yq template://default '.images[].location'

The example command is equivalent to using an external yq command like this:
    limactl template copy --fill template://default - | yq '.images[].location'

Usage:
  limactl template yq TEMPLATE EXPR [flags]
...
```